librocksdb_sys/crocksdb/c.cc

```diff
@@ -2789,6 +2789,54 @@ void crocksdb_readoptions_set_ignore_range_deletions(
   opt->rep.ignore_range_deletions = v;
 }
 
+struct TableFilter {
```

Comment:
Can we create another struct like `TableFilterContext` to wrap the related context, then put a `shared_ptr` inside `TableFilter`?